papergres.go

```diff
@@ -533,22 +545,27 @@ func goToSQLName(name string) string {
 }
 
 // insertArgs creates the insert arg slice for an object
-func insertArgs(obj interface{}) []interface{} {
-	final, _ := prepareFields(obj)
+func insertArgs(obj interface{}, withPK bool) []interface{} {
+	final, _ := prepareFields(obj, withPK)
 	args := make([]interface{}, len(final))
 	for i, f := range final {
 		args[i] = f.Value
 	}
 	return args
 }
 
-// prepareFields performs necessary transformations for the insert statement
-func prepareFields(obj interface{}) (nfields []*Field, primary *Field) {
+// prepareFields performs necessary transformations for the insert statement.
```

Comment:
See above regarding comments

Reply:
Same as above. Line comments are the norm.